projects/objectnav_baselines/experiments/ithor/objectnav_ithor_base.py

```diff
@@ -92,7 +92,7 @@ def machine_params(self, mode="train", **kwargs):
             sampler_devices = self.TRAIN_GPU_IDS
             render_video = False
         elif mode == "valid":
-            nprocesses = 15
+            nprocesses = 1
```

Comment:
Is this what we want?

Reply:
Validation only works with 1 GPU (as per jordi) and I also tested this myself and using more than 1 GPU broke things for me. I tried using 15 processes here on a single GPU and that also seemed to break things. I definitely agree that using 15 processes here would be better though, but I was not sure if Jordi was saying that we can only use 1 process on 1 GPU or n processes on 1 GPU.